app/controllers/customers_controller.rb

```diff
@@ -25,7 +25,7 @@ def create
     @customer = Customer.new(customer_params)
     if @customer.save
       flash.notice = "The customer record was created successfully."
-      redirect_to @customer
+      redirect_to root_url
```

Comment:
I know that the root currently redirects to the customers#index, but it'd make more sense to redirect to `customers_path` because I think that's really what you want, and it will still work if you root ever changes.

Reply:
great, fixed it and agree that your redirection path logic is more correct